extensions/msal-node-extensions/package.json

```diff
@@ -37,15 +37,15 @@
   },
   "module": "dist/test.esm.js",
   "dependencies": {
-    "bindings": "^1.5.0",
     "@azure/msal-common": "^1.6.3",
-    "keytar": "6.0.1",
+    "bindings": "^1.5.0",
+    "keytar": "^7.0.0",
```

Comment:
@sameerag This shouldn't break anything, but worth testing: https://github.com/atom/node-keytar/releases/tag/v7.0.0